src/ResourceManager/DataLakeStore/Commands.DataLakeStore/Models/GlobalMembers.cs

```diff
@@ -0,0 +1,69 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Commands.DataLakeStore.Models
+{
+    /// <summary>
+    ///     A Static class containing global members for these cmdlets.
+    /// </summary>
+    public static class GlobalMembers
+    {
+        /// <summary>
+        ///     The binary file extension list
+        /// </summary>
+        public static readonly HashSet<string> BinaryFileExtension = new HashSet<string>
```

Comment:
This is fine to detect if file is binary or not though it may worth doing this in more generic way, take a look in [this thread](http://stackoverflow.com/questions/58510/using-net-how-can-you-find-the-mime-type-of-a-file-based-on-the-file-signature) it may has some useful hints